src/main/scala/Switcher.scala

```diff
@@ -103,10 +103,10 @@ class TLSwitcher(
 
   val device = new SimpleDevice("switcher", Seq("ucb-bar,switcher"))
 
-  val innode = TLManagerNode(Seq.fill(inPortN) {
+  val innode = TLManagerNode(Seq.tabulate(inPortN) { i =>
     TLManagerPortParameters(
       Seq(TLManagerParameters(
-        address    = address,
+        address    = Seq(address(i)),
```

Comment:
This is the only place where it looks like behavior might have changed. Now each `TLManagerParameters` object is getting a 1-long `Seq` of `AddressSet` rather than a `Seq` of all `AddressSets`. The new style seems more correct- how did it work before? Is it because we always gave it a 1-long `AddressSet`?

Reply:
We used to have an mbus per memory channel and do a loop through them such that each connection was essentially for a 1-long `Seq[AddressSet]`